packages/contracts-bedrock/test/Safe/LivenessGuard.t.sol

```diff
@@ -245,6 +245,8 @@ contract LivenessGuard_FuzzOwnerManagement_Test is StdCheats, StdUtils, Liveness
             privateKeys[ownerAddrs[i]] = ownerkeys[i];
         }
 
+        // Override the saltNonce to ensure prevent a create2 collision.
+        saltNonce = uint256(keccak256(bytes("LIVENESS GUARD OWNER MANAGEMENT TEST")));
         // Create the new safe and register the guard.
         SafeInstance memory safeInstance = _setupSafe(ownerkeys, threshold);
```

Comment:
This shadows a storage variable of the same name, is that intentional? I know it wasn't changed in this PR, but would love either a quick rename to avoid shadowing, or delete the `SafeInstance memory` so it's now using the storage variable.

Reply:
The intent is to set that same storage variable, which I think it does (rather than shadow it?). 

The issue this PR seeks to address is that there needs to be a way to change the `saltNonce` or else you can only call `_setupSafe()` once without causing a Create2 collision.

Reply:
Sorry, to clarify, agreed the `saltNonce` is implemented correctly. I was commenting on an unchanged line, since VSCode flagged it to me. This line:
```solidity
SafeInstance memory safeInstance = _setupSafe(ownerkeys, threshold);
```
shadows a storage var named `safeInstance`

Reply:
Oh, I see: 070c141